test/model.test.js

```diff
@@ -4336,7 +4326,6 @@ describe('Model', function() {
   it('save max bson size error with buffering (gh-3906)', async function() {
     this.timeout(10000);
     const db = start({ noErrorListener: true });
```

Comment:
from what i can tell, in this file / tests the connections do not get properly closed if the test is not successful (which `connectionsToClose` would solve)